src/core/normalizer.coffee

```diff
@@ -57,9 +57,10 @@ class Normalizer
         node.removeAttribute(attribute)
     )
     # Chrome turns <b> into style in some cases
-    if (node.style.fontWeight == 'bold')
+    if (node.style.fontWeight == 'bold' or node.style.fontWeight > 500)
```

Comment:
Can you change this to > 400? Technically a font-weight of 500 is also considered bolded.